### PR TITLE
Analysis of meals on speech days

### PR DESCRIPTION
**Pull request by fgrehm**
_Related to GH-137_

This also has the the bonus of setting up [serenata_toolbox](https://github.com/datasciencebr/serenata-de-amor/issues/143). It is only used for this notebook ATM but I can look into updating other stuff to make use of it at another day this or next week 🤘

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
+-e git+https://github.com/datasciencebr/serenata-toolbox.git@6126436#egg=serenata-toolbox
```
Comment (Irio) [inline requirements.txt]: @cuducos You [recently removed](https://github.com/datasciencebr/rosie/commit/9414eb3195ea7c4b9c8d9b8d073f3bbd078482e2) the `-e` from Rosie. Was for not saving it inside the project folder, right?

If that's the case, can you change for the version after @cuducos' commit?

Comment (cuducos) [inline requirements.txt]: Yep. `-e` means editable mode, so it copy the package source code to your project. We don't need that here ; )

Comment (fgrehm) [inline requirements.txt]: Sure! I'll get that changed later on today 👍

File: requirements.txt [hunk 2]
```diff
@@ -1,2 +1,4 @@
+git+https://github.com/datasciencebr/serenata-toolbox.git@6126436#egg=serenata-toolbox
```
Comment (fgrehm) [inline requirements.txt]: @Irio @cuducos I've removed the `-e` but was wondering if you guys had any thoughts on keeping it locked to a version like I did here? I like that it makes things reproductible and makes it clear when we expect people to update the package on their local installations but I noticed you guys didn't do it on rosie 💭

Comment (cuducos) [inline requirements.txt]: Very very personal opinion here: locking the version surely is a good thing, but I think it should be done in a neat way. It's a good point you raise here, I don't think we've pondered on it in this context… Anyway: a neat way would be to use GitHug tags, or (I prefer) uploading it to PyPI.

Comment (fgrehm) [inline requirements.txt]: yeah, uploading to pypi would be ideal, but a git tag might be the easiest way to move forward for now 💭

Comment (cuducos) [inline requirements.txt]: @irio: Any ideas? If the silence remains (_drama_ 👑 ) I' upload it to PyPI next week hahaha…

Comment (Irio) [inline requirements.txt]: @cuducos  :shipit:

**Comment by fgrehm** [comment]
@Irio @cuducos since master already have serenata-toolbox, I guess this can be merged now

**Pull request merged**